src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -451,20 +451,58 @@ inline void rpp_load48_u8pkd3_to_u8pln3(Rpp8u *srcPtr, __m128i *px)
     px[2] = _mm_shuffle_epi8(_mm_unpacklo_epi8(pxSrc[6], pxSrc[7]), pxMaskRGB);    /* unpack 8 lo-pixels of pxSrc[6] and pxSrc[7] to get B01-16 */
 }
 
+inline void rpp_load96_u8pkd3_to_u8pln3(Rpp8u *srcPtr, __m256i *px)
+{
+    __m256i pxSrc[8];
+    __m256i pxMask = _mm256_castsi128_si256(_mm_setr_epi8(0, 3, 6, 9, 1, 4, 7, 10, 2, 5, 8, 11, 12, 13, 14, 15));
```

Comment:
Quite many loads and stores added in this PR.
Please do recheck if no similar ones are already present in this rpp_cpu_simd file from both bitwiseAND and bitwiseOR implementations?
Was presuming bitwiseXOR should only change the compute and not the loads/stores?

Reply:
With regards to the load and store functions, the previous logical operation implementations (for bitwise AND and bitwise OR) use 128 bit SSE Vector (48 load/store) for u8 and i8 bit depths
Here we have updated the load and store functions for exclusive or to use 256 bit AVX Vector for u8 and i8 bit depths also
Regarding the same, initially discussions were done with Sampath and he was of the opinion that if we use 96 load/store and if there is no dip in performance for samples tested, then we could persist with the same. 
Based on perf numbers for PLN3/PLN3 and PLN3 to PKD3 it was decided to go ahead with 96 load/store

Hence for u8 and i8, load/store routines are updated for different bit depths

Reply:
Instead of rpp_load96_u8pln3_to_u8pln3 , rpp_load96_u8_avx was used which was already used as part of  tensor_max.hpp. There was no equivalent for i8pln3_to_u8pln3, so retained the same